x-pack/plugins/endpoint/scripts/resolver_generator.ts

```diff
@@ -29,7 +29,7 @@ async function main() {
     alertIndex: {
       alias: 'ai',
       describe: 'index to store alerts in',
-      default: '.alerts-endpoint-000001',
+      default: 'events-endpoint-1',
```

Comment:
@bkimmel I agree that we should probably keep this as `events-endpoint-1` since that's what the app is looking for in master.

@elastic/endpoint-response are you ok with this?